packages/mdc-textfield/foundation.js

```diff
@@ -204,7 +206,11 @@ class MDCTextFieldFoundation extends MDCFoundation {
     const labelScale = isDense ? numbers.DENSE_LABEL_SCALE : numbers.LABEL_SCALE;
     const labelWidth = this.adapter_.getLabelWidth() * labelScale;
     const isRtl = this.adapter_.isRtl();
-    this.adapter_.updateOutlinePath(labelWidth, isRtl);
+    if (activateNotch) {
+      this.adapter_.notchOutline(labelWidth, isRtl);
+    } else {
+      this.adapter_.notchOutline(0);
```

Comment:
setting the labelWidth = 0 in order to simulate "closing the notch" is unintuitive. Just add another method that explicitly closes the notch